Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -813,13 +824,16 @@ internal class PythonProjectNode :
                 foreach (var fact in InterpreterFactories) {
                     if (!RemoveFirst(remaining, n => !n._isGlobalDefault && n._factory == fact)) {
                         bool isProjectSpecific = vsProjectContext.IsProjectSpecific(fact.Configuration);
+                        bool canRemove = !this.IsAppxPackageableProject(); // Do not allow change python enivronment for UWP
                         node.AddChild(new InterpretersNode(
                             this,
                             fact,
                             isInterpreterReference: !isProjectSpecific,
                             canDelete:
                                 isProjectSpecific &&
-                                Directory.Exists(fact.Configuration.PrefixPath)
+                                Directory.Exists(fact.Configuration.PrefixPath),
+                            isGlobalDefault:false,
+                            canRemove:canRemove
```

Comment:
I think this is probably better expressed as a new InterpreterUIMode value, but we can fix that up later.